### DISCUSSION TITLE
Cannot redeem USDC from BSC to Solana

### DISCUSSION THREAD

**Discussion by EXOTICFOLD** [wormhole-foundation/wormhole#616] [Problems / bugs] [answered]
I'm doing everything correctly. Have been following the steps in wormholebridge.com/#/transfer from Step 1 through the last but when I press "Redeem” button, every time they charge 2 or 3 fees then at last it keeps giving me error messages “Transaction was not confirmed in 30.02 seconds..." The USDC just hasn't arrived my Solana wallet after 10+ attempts and 2 hours of wait. What should I do?

**Comment by evan-gray** [comment, marked as answer]
Solana was suffering from some congestion earlier today (and may still be recovering). The current UI requires a series of transactions to complete successfully, so waiting for a less congested time may be the best approach. If / when the suggested fix in #574 is implemented it would help this situation as well.
